inc/class_core.php

```diff
@@ -519,6 +521,8 @@ public function get_asset_url($path = '', $use_cdn = true)
 			$url = $path;
 		}
 
+		$plugins->run_hooks('get_asset_url_end', $url);
```

Comment:
We can always remove it as it's not crucial, but it could be nice to have for some developers.